src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -168,7 +173,7 @@ public static void setupCodeSyncRepo(Project project, String repoPath, String re
                 // That is why this boolean is placed here.
                 boolean shouldSyncRepo = false;
                 // Do not ask user to sync repo, if it is already in progress.
-                if (!reposBeingSynced.contains(repoPath)) {
+                if (!reposBeingSynced.contains(repoPath) && !Utils.isIndividualFileOpen(project)) {
```

Comment:
I do not think we need this here, it will cause more bugs in this place.

Reply:
Yes, removed it.